static/coffee/modals.coffee

```diff
@@ -90,13 +90,25 @@ modals.controller 'NewCaseModalController', ['$scope', '$uibModalInstance', 'sum
 #=====================================================================
 # Assign to partner modal
 #=====================================================================
-modals.controller 'AssignModalController', ['$scope', '$uibModalInstance', 'title', 'prompt', 'partners', ($scope, $uibModalInstance, title, prompt, partners) ->
+modals.controller 'AssignModalController', ['$scope', '$uibModalInstance', 'title', 'prompt', 'partners', 'users', 'UserService', ($scope, $uibModalInstance, title, prompt, partners, users, UserService) ->
   $scope.title = title
   $scope.prompt = prompt
   $scope.partners = partners
-  $scope.fields = { assignee: partners[0] }
 
-  $scope.ok = () -> $uibModalInstance.close($scope.fields.assignee)
+  user_list_from_users = (users) ->
+    userlist = [{id: null, name: "Anyone"}].concat(users)
+    console.log(userlist)
```

Comment:
Resolved in https://github.com/praekelt/casepro/pull/69/commits/550735c1eace1f52d030ccf37695bc62c2d05068